articles/clients/client-grant-types.md

```diff
@@ -69,6 +69,10 @@ Public Clients, indicated by the `token_endpoint_auth_method` flag set to `none`
 * `authorization_code`;
 * `refresh_token`.
 
+::: note
```

Comment:
@khorne3 We keep referring to this `token_endpoint_auth_method` in this doc but we don't explain what it is or how you change it.